### ISSUE TITLE
Evaluation / training code

### ISSUE THREAD

**Issue by helen1c** [wimmerth/anyup#8]
Hi @wimmerth, cool work, congrats. :-)

I’m very interested in building on top of your work, and would love to know more about the evaluation and training setup.
Do you have plans to release the corresponding scripts, especially those needed to reproduce the results from your paper?
If so, could you share an approximate timeline for their release?

Thanks a lot!

Best, Ivan

**Comment by wimmerth** [comment]
Hi, great to see your interest in AnyUp. Of course, we are planning the release of the training code!
Expect it to happen within the next two weeks :)

Regarding the evaluation, we followed the setups of [Probe3D](https://github.com/mbanani/probe3d) and [JAFAR](https://github.com/PaulCouairon/JAFAR), as described in our paper. In the probe training of JAFAR, be aware that we moved the LR scheduler `step()` function outside of the inner training step loop to be called once per epoch and not once per processed batch. For more details on the probe training with JAFAR, see the [following issue](https://github.com/PaulCouairon/JAFAR/issues/5)

**Comment by wimmerth** [comment]
I just uploaded the training code.
